modules/git/git.go

```diff
@@ -15,7 +15,7 @@ import (
 
 	"code.gitea.io/gitea/modules/process"
 
-	"github.com/hashicorp/go-version"
+	"github.com/6543/go-version"
```

Comment:
Instead of updating all of these references, could you instead add a replace call to the go.mod so that when (if?) the PR is accepted upstream then we could just remove the replace

Reply:
@techknowlogick as in Discord ...

I get errors like `replace github.com/hashicorp/go-version: version "1.2.1" invalid: unknown revision 1.2.1`
If you have more luck I'm happy if you could push your solution :/